<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml">
  <head>
    <meta charset="utf-8" />
    <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
    <meta name="generator" content="pandoc" />
    <meta name="viewport" content="width=device-width, initial-scale=1">

    <meta name="author" content="Ming Tang" />
    <meta name="date" content="2020-06-24" />
    <title>Ming Tang’s resume</title>

    
        <link href="Tang_Ming_cv_files/font-awesome-5.1.0/css/all.css" rel="stylesheet" />
    <link href="Tang_Ming_cv_files/font-awesome-5.1.0/css/v4-shims.css" rel="stylesheet" />
    <link href="Tang_Ming_cv_files/paged-0.10/css/resume.css" rel="stylesheet" />
    <script src="Tang_Ming_cv_files/paged-0.10/js/config.js"></script>
    <script src="Tang_Ming_cv_files/paged-0.10/js/paged.js"></script>
    <script src="Tang_Ming_cv_files/paged-0.10/js/hooks.js"></script>
    
    
  </head>
  <body>

<style type="text/css">
.pagedjs_page:not(:first-of-type) {
  background: white;
}

.pagedjs_page:not(:first-of-type) {
  --sidebar-width: 0rem;
  --sidebar-background-color: #ffffff;
  --main-width: calc(var(--content-width) - var(--sidebar-width));
  --decorator-horizontal-margin: 0.2in;
}
</style>
<div id="aside" class="section level1">
<h1>Aside</h1>
<div id="contact" class="section level2">
<h2>Contact Info</h2>
<ul>
<li><i class="fa fa-envelope"></i> <a href="mailto:tangming2005@gmail.com" class="email">tangming2005@gmail.com</a></li>
<li><i class="fa fa-github"></i> <a href="https://github.com/crazyhottommy">github.com/crazyhottommy</a></li>
<li><i class="fa fa-phone"></i> +1 352-226-9384</li>
<li>website: <a href="https://divingintogeneticsandgenomics.rbind.io/">divingintogeneticsandgenomics.rbind.io/</a></li>
<li>For more information, please contact me via email.</li>
</ul>
</div>
<div id="skills" class="section level2">
<h2>Skills</h2>
<ul>
<li><p>Extensive experience with next generation sequencing data analysis and pipeline workflow.</p></li>
<li><p>Experienced in Single-cell data analysis.</p></li>
<li><p>Highly skilled in R, Bash and Python.</p></li>
<li><p>Great communication skills.</p></li>
</ul>
</div>
<div id="disclaimer" class="section level2">
<h2>Disclaimer</h2>
<p>This resume was made with the R<br />
package <a href="https://github.com/rstudio/pagedown"><strong>pagedown</strong></a>.</p>
<p>Last updated on 2020-06-24.</p>
</div>
</div>
<div id="main" class="section level1">
<h1>Main</h1>
<div id="title" class="section level2">
<h2>Ming Tang</h2>
<p><strong>Senior Scientist</strong><br />
<strong>Department of Data Science</strong><br />
<strong>Dana-Farber Cancer Institute</strong><br />
<strong>Boston, MA, 02215</strong></p>
</div>
<div id="employment" class="section level2" data-icon="address-book">
<h2>Employment</h2>
<div id="postdoctoral-fellow" class="section level3">
<h3>Postdoctoral Fellow</h3>
<p>MD Anderson Cancer Center</p>
<p>TX, USA</p>
<p>2015 - 2017</p>
</div>
<div id="research-scientist" class="section level3">
<h3>Research Scientist</h3>
<p>MD Anderson Cancer Center</p>
<p>TX, USA</p>
<p>2017 - 2018</p>
</div>
<div id="senior-bioinformatics-scientist" class="section level3">
<h3>Senior Bioinformatics Scientist</h3>
<p>Harvard FAS informatics</p>
<p>MA, USA</p>
<p>2018 - 2020</p>
</div>
<div id="senior-scientist" class="section level3">
<h3>Senior Scientist</h3>
<p>Dana-Farber Cancer Institute</p>
<p>MA, USA</p>
<p>2020-</p>
</div>
</div>
<div id="education" class="section level2" data-icon="graduation-cap" data-concise="true">
<h2>Education</h2>
<div id="shanghai-jiaotong-university" class="section level3">
<h3>Shanghai Jiaotong University</h3>
<p>B.S. in Biotechnology</p>
<p>Shanghai, China</p>
<p>2008</p>
</div>
<div id="university-of-florida" class="section level3">
<h3>University of Florida</h3>
<p>Ph.D. in Genetics and Genomics</p>
<p>FL, USA</p>
<p>2014</p>
</div>
</div>
<div id="honors-and-awards" class="section level2">
<h2>Honors and awards</h2>
<div id="peoples-scholarship" class="section level3">
<h3>People’s Scholarship</h3>
<p>Shanghai Jiaotong University</p>
<p>China</p>
<p>2006</p>
</div>
<div id="national-stipend-for-academic-excellence" class="section level3">
<h3>National Stipend for Academic Excellence</h3>
<p>Shanghai Jiao Tong University</p>
<p>China</p>
<p>2007</p>
</div>
<div id="national-scholarship-for-academic-excellence" class="section level3">
<h3>National Scholarship for Academic Excellence</h3>
<p>Shanghai Jiao Tong University</p>
<p>China</p>
<p>2007</p>
</div>
<div id="outstanding-international-student" class="section level3">
<h3>Outstanding International Student</h3>
<p>University of Florida</p>
<p>USA</p>
<p>2012</p>
</div>
<div id="genetics-genomics-program-travel-grant" class="section level3">
<h3>Genetics &amp; Genomics program Travel Grant</h3>
<p>University of Florida</p>
<p>USA</p>
<p>2013</p>
</div>
<div id="next-generation-sequencing-course-certificate" class="section level3">
<h3>Next Generation Sequencing course certificate</h3>
<p>Michigan State University</p>
<p>USA</p>
<p>2014</p>
</div>
<div id="summer-institutes-in-statistics-for-big-data-scholarship" class="section level3">
<h3>Summer Institutes in Statistics for Big Data Scholarship</h3>
<p>University of Washington</p>
<p>USA</p>
<p>2015</p>
</div>
<div id="usc-bd2k-data-science-road-trip-fellow" class="section level3">
<h3>USC BD2K Data Science Road-Trip fellow</h3>
<p>University of Southern California</p>
<p>USA</p>
<p>2018</p>
</div>
<div id="bioconductor-annual-meeting-travel-award" class="section level3">
<h3>Bioconductor annual meeting Travel award</h3>
<p>Bioconductor</p>
<p>NYC, USA</p>
<p>2019</p>
</div>
<div id="top-75-bioinformatics-blogs-and-websites-for-bioinformaticians" class="section level3">
<h3>Top 75 Bioinformatics Blogs and Websites for Bioinformaticians</h3>
<p>feedspot.com</p>
<p>USA</p>
<p>2019</p>
</div>
</div>
<div id="publications-for-co-first-author" class="section level2" data-icon="file">
<h2>Publications (* for co-first author)</h2>
<div id="restraint-of-angiogenesis-by-zinc-finger-transcription-factor-ctcf-dependent-chromatin-insulation." class="section level3">
<h3>Restraint of angiogenesis by zinc finger transcription factor CTCF-dependent chromatin insulation.</h3>
<p><em>Proc Natl Acad Sci USA</em>. 2011 Sep 13;108(37):15231-6. PubMed PMID: 21896759</p>
<p>N/A</p>
<p>2011</p>
<p><strong>Tang M</strong>, Chen B, Lin T, Li Z, Pardo C, et al.</p>
</div>
<div id="ctcf-dependent-chromatin-insulator-as-a-built-in-attenuator-of-angiogenesis." class="section level3">
<h3>CTCF-dependent chromatin insulator as a built-in attenuator of angiogenesis.</h3>
<p><em>Transcription</em>. 2012 Mar-Apr;3(2):73-7. PubMed PMID: 22414750; PubMed Central PMCID: PMC3337828.</p>
<p>N/A</p>
<p>2012</p>
<p>Lu J and <strong>Tang M</strong>.</p>
</div>
<div id="the-malignant-brain-tumor-mbt-domain-protein-sfmbt1-is-an-integral-histone-reader-subunit-of-the-lsd1-demethylase-complex-for-chromatin-association-and-epithelial-to-mesenchymal-transition." class="section level3">
<h3>The malignant brain tumor (MBT) domain protein SFMBT1 is an integral histone reader subunit of the LSD1 demethylase complex for chromatin association and epithelial-to-mesenchymal transition.</h3>
<p><em>J Biol Chem</em> 2013 Sep 20;288(38):27680-91. PubMed PMID: 23928305;PubMed Central PMCID: PMC3779763.</p>
<p>N/A</p>
<p>2013</p>
<p><strong>Tang M</strong>, Shen H, Jin Y, Lin T, Cai Q, et al</p>
</div>
<div id="trim29-suppresses-twist1-and-invasive-breast-cancer-behavior." class="section level3">
<h3>TRIM29 suppresses TWIST1 and invasive breast cancer behavior.</h3>
<p><em>Cancer Res</em>. 2014 Sep 1;74(17):4875-87. PubMed PMID: 24950909.</p>
<p>N/A</p>
<p>2014</p>
<p>Ai L, Kim WJ, Alpay M, <strong>Tang M</strong>, Pardo CE, et al.</p>
</div>
<div id="genomic-and-proteomic-analysis-of-transcription-factor-tfii-i-reveals-insight-into-the-response-to-cellular-stress." class="section level3">
<h3>Genomic and proteomic analysis of transcription factor TFII-I reveals insight into the response to cellular stress.</h3>
<p><em>Nucleic Acids Res</em>. 2014;42(12):7625-41. PubMed <a href="PMID:24875474" class="uri">PMID:24875474</a>; PubMed Central PMCID: PMC4081084.</p>
<p>N/A</p>
<p>2014</p>
<p>Fan AX, Papadopoulos GL, Hossain MA, Lin IJ, Hu J, <strong>Tang M</strong>, et al.</p>
</div>
<div id="mof-acetylates-the-histone-demethylase-lsd1-to-suppress-epithelial-to-mesenchymal-transition." class="section level3">
<h3>MOF Acetylates the Histone Demethylase LSD1 to Suppress Epithelial-to-Mesenchymal transition.</h3>
<p><em>Cell Reports</em>. 2016.PMID: 27292636</p>
<p>N/A</p>
<p>2016</p>
<p>Luo HC, Shenoy AK, Li X, Jin Y, Jin L, Cai Q, <strong>Tang M</strong>, Liu Y, Chen H et.al.</p>
</div>
<div id="a-molecular-take-on-malignant-rhabdoid-tumors." class="section level3">
<h3>A Molecular Take on Malignant Rhabdoid Tumors.</h3>
<p><em>Trends in Cancer</em> 2016. May;2(5):217-218</p>
<p>N/A</p>
<p>2016</p>
<p><strong>Tang M</strong> and Verhaak R.</p>
</div>
<div id="suppression-of-type-i-ifn-signaling-in-tumors-mediates-resistance-to-anti-pd-1-treatment-that-can-be-overcome-by-radiotherapy." class="section level3">
<h3>Suppression of type I IFN signaling in tumors mediates resistance to anti-PD-1 treatment that can be overcome by radiotherapy.</h3>
<p><em>Cancer Res</em>. 2016. Nov 7. pii: canres.3142.2015.</p>
<p>N/A</p>
<p>2016</p>
<p>Wang X, Schoenhals JE, Li A, Valdecanas DR, Ye H, Zhang F, Tang C, <strong>Tang M</strong>,
Liu CG, LiuX, Krishnan S, Allison JP, Sharma P, Hwu P, Komaki R, Overwikjk WW,
Gomez DR, Chang JY, Hahn SM, Cortez Mam Welsh JW</p>
</div>
<div id="epithelial-to-mesenchymal-transition-confers-pericyte-properties-on-cancer-cells." class="section level3">
<h3>Epithelial-to-mesenchymal transition confers pericyte properties on cancer cells.</h3>
<p><em>Journal of Clinical Investigation</em>. 2016. Nov 1;126(11):4174-4186. doi: 10.1172/JCI86623</p>
<p>N/A</p>
<p>2016</p>
<p>Shenoy AK, Jin Y, Luo H, <strong>Tang M</strong>, Pampo C, Shao R, Siemann DW, Wu L,
Heldermon C, Law BK, Chang LJ, Lu J.</p>
</div>
<div id="systematic-analysis-of-telomere-length-telomerase-and-telomere-maintenance-across-31-cancer-types." class="section level3">
<h3>Systematic Analysis of Telomere Length, Telomerase and Telomere Maintenance across 31 cancer types.</h3>
<p><em>Nature Genetics</em>. 2017. <a href="doi:10.1038/ng.3781" class="uri">doi:10.1038/ng.3781</a></p>
<p>N/A</p>
<p>2017</p>
<p>Barthel FP, Wei W, <strong>Tang M</strong>, Martinez-Ledesma E, Hu X, Amin S, Seth S, Song X,
Lichtenberg T, Hu J, Zhang J, Zheng S, Verhaak R.</p>
</div>
<div id="synthetic-essentiality-of-chromatin-remodeling-factor-chd1-in-pten-deficient-cancer" class="section level3">
<h3>Synthetic essentiality of chromatin remodeling factor CHD1 in PTEN-deficient cancer</h3>
<p><em>Nature</em>. 2017. <a href="doi:10.1038/nature21357" class="uri">doi:10.1038/nature21357</a></p>
<p>N/A</p>
<p>2017</p>
<p>Zhao D, Lu X, Wang G, Lan Z, Liao W, Li J, Liang X Chen J, Shah S, Shang X,
<strong>Tang M</strong>, Deng P, Dey P, Chakravarti D, Chen P, Spring D, Navone N, Troncoso P,
Zhang j, Wang YA, DePinho RA</p>
</div>
<div id="integrative-analysis-identifies-four-molecular-and-clinical-subsets-in-uveal-melanoma" class="section level3">
<h3>Integrative Analysis identifies Four Molecular and Clinical Subsets in Uveal Melanoma</h3>
<p><em>Cancer Cell</em>. 2017.32:2:p204-220.e15</p>
<p>N/A</p>
<p>2017</p>
<p>Robertson AG, Shih J, Yau C, Gibb EA, Mungall KL, Hess JM, Uzunangelov V,
Walter V, Danilova L, Lichtenberg TM, Kucherlapati M, Kimes PK, <strong>Tang M</strong>, Penson
A, Babur O, Bristow CA, Hoadley KA, Iype L, Chang MT, Oba J, The Cancer
Genome Atlas (TCGA) Network, Cherniack AD, Benz C, Verhaak R, Griewank KG,
Felau I, Zenklusen JC, Gershenwald JE, Schoenfield L, Lazar AJ, Abdel-Rahman
M, Roman-Roman S, Stern, MH, Cebulla CM, Williams MD, Jager MJ, Coupland S,
Esmaeli B, Kandoth C, Woodman SE</p>
</div>
<div id="tfii-i-interacts-with-e2f-transcription-factors-and-regulates-their-association-with-the-cooccupied-atf3-gene-locus." class="section level3">
<h3>TFII-I interacts with E2F transcription factors and regulates their association with the cooccupied ATF3 gene locus.</h3>
<p><em>Jounral of Cellular Biochemistry</em>. 2017. <a href="Doi:10.1002/jcb.26235" class="uri">Doi:10.1002/jcb.26235</a></p>
<p>N/A</p>
<p>2017</p>
<p>Fan A, Aryan M, Shen Y, Hossain M, <strong>Tang M</strong>, Lu J, Strouboulis J, Bungert J.</p>
</div>
<div id="tumorfusions-an-integrative-resource-for-reporting-cancerassociated-transcript-fusion-in-33-tumor-types." class="section level3">
<h3>TumorFusions: an integrative resource for reporting cancerassociated transcript fusion in 33 tumor types.</h3>
<p><em>Nucleic Acids Research</em>. 2017.Doi:10.1093/nar/gkx1018/4584571</p>
<p>N/A</p>
<p>2017</p>
<p>Hu X, Wang Q, <strong>Tang M</strong>, Barthel FP, Amin S, Yoshihara K, Lang FM, Lee SH, Zheng S, Verhaak R.</p>
</div>
<div id="an-integrated-platform-for-genome-wide-mapping-of-chromatin-states-using-highthroughput-chip-sequencing-in-tumor-tissues" class="section level3">
<h3>An Integrated Platform for Genome-wide Mapping of Chromatin States Using Highthroughput ChIP-sequencing In Tumor Tissues</h3>
<p><em>Journal of Visualized Experiments</em>. 2018.(134),e56972,<a href="doi:10.3791/56972" class="uri">doi:10.3791/56972</a>.</p>
<p>N/A</p>
<p>2018</p>
<p>Terranova C, <strong>Tang M</strong>, Orouji E, Maitituoheti M, Raman A, Amin S, Liu Z, Rai K.</p>
</div>
<div id="the-tandem-duplicator-phenotype-is-a-prevalent-genome-wide-cancer-configuration-driven-by-distinct-gene-mutations" class="section level3">
<h3>The Tandem Duplicator Phenotype Is a Prevalent Genome-Wide Cancer Configuration Driven by Distinct Gene Mutations</h3>
<p><em>Cancer Cell</em>. 2018. 13;34(2)197-210.e5 <a href="doi:10.1016/j.ccell.2018.06.008" class="uri">doi:10.1016/j.ccell.2018.06.008</a></p>
<p>N/A</p>
<p>2018</p>
<p>Menghi F, Barthel FP, Yadav V, <strong>Tang M</strong>, Ji B, Tang Z, Cater GW, Ruan Y, Scully R,
Verhaak R, Jonkers J, Liu E.</p>
</div>
<div id="positive-regulation-of-transcription-by-human-zmynd8-through-its-association-with-ptefb-complex." class="section level3">
<h3>Positive Regulation of Transcription by Human ZMYND8 through Its Association with PTEFb Complex.</h3>
<p><em>Cell Reports</em>. 2018. Aug21;24(8):21441-2154.e6 <a href="doi:10.1016/j.celrep.2018.07.064" class="uri">doi:10.1016/j.celrep.2018.07.064</a></p>
<p>N/A</p>
<p>2018</p>
<p>Ghosh K, <strong>Tang M</strong>, Kumari N, Nandy A, Basu S, Mall DP, Rai K, Biswas D</p>
</div>
<div id="histone-deacetylase-inhibitor-targets-cd123cd47-positive-cells-and-reverse-chemoresistance-phenotype-in-acute-myeloid-leukemia." class="section level3">
<h3>Histone deacetylase inhibitor targets CD123/CD47-positive cells and reverse chemoresistance phenotype in acute myeloid leukemia.</h3>
<p><em>Leukemia</em>. 2018. <a href="Doi:1038/s41375-018-0279-6" class="uri">Doi:1038/s41375-018-0279-6</a>.</p>
<p>N/A</p>
<p>2018</p>
<p>Yan B, Chen Q, Shimada K, <strong>Tang M</strong>, Li H, Gurumurthy A, Khoury JD, Xu B, Huang S, Qiu Y</p>
</div>
<div id="landscape-of-egfr-dependent-and--independent-resistance-mechanisms-to-osimertinib-and-continuation-therapy-beyond-progression-in-egfr-mutant-nsclc." class="section level3">
<h3>Landscape of EGFR-Dependent and -Independent Resistance Mechanisms to Osimertinib and Continuation Therapy Beyond Progression in EGFR-Mutant NSCLC.</h3>
<p><em>Clin Cancer Res</em>. 2018 Dec 15;24(24):6195-6203. doi: 10.1158/1078-0432.CCR-18-1542.</p>
<p>N/A</p>
<p>2018</p>
<p>Le X, Puri S, Negrao M, Nilsson M, Robichaux J, Boyle T, Hicks J, Lovinger K, Roarty E,
Rinsurongkawong W,<strong>Tang M</strong>, Sun H, Elamin Y, Lacerda L, Lewis J, Roth J, Swisher S, Lee J, William W, Glisson B,
Zhang J, Papadimitrakopoulou V, Gray J, Heymach J.</p>
</div>
<div id="kras-irf2-axis-drives-immune-suppression-and-immune-therapy-resistance-in-colorectal-cancer." class="section level3">
<h3>KRAS-IRF2 Axis Drives Immune Suppression and Immune Therapy Resistance in Colorectal Cancer.</h3>
<p><em>Cancer Cell</em>. 2019 Apr 15;35(4):559-572.e7. doi: 10.1016/j.ccell.2019.02.008.</p>
<p>N/A</p>
<p>2019</p>
<p>Liao W, Overman M, Boutin A, Shang X, Zhao D, Dey P, Li J, Wang G, Lan Z, Li J, <strong>Tang M</strong>,
Jiang S, Ma X, Chen P, Katkhuda R, Korphaisarn K, Chakravarti D, Chang A, Spring D, Chang Q, Zhang J,
Maru D, Maeda D, Zebala J, Kopetz S, Wang Y, Depinho R.</p>
</div>
<div id="distinct-co-acquired-alterations-and-genomic-evolution-during-tki-treatment-in-non-small-cell-lung-cancer-patients-with-or-without-acquired-t790m-mutation" class="section level3">
<h3>Distinct Co-Acquired Alterations and Genomic Evolution During TKI Treatment in Non-Small-Cell Lung Cancer Patients With or Without Acquired T790M Mutation</h3>
<p><em>Oncogene</em>. 2020 Feb;39(9):1846-1859. doi: 10.1038/s41388-019-1104-z.</p>
<p>N/A</p>
<p>2019</p>
<p>Jin Y, Bao H, Le X, Fan X, <strong>Tang M</strong>, Shi X, Zhao J, Yan J, Xu Y, Quek K, Elamin Y, Zhan J, Futreal A, Wistuba I, Heymach J, Lou G, Shao L, He Q, Lin C, Wu X, Shao Y, Wang X, He J, Chen Y, Stebbing J, Chen M, Zhang J, Yu X.</p>
</div>
<div id="lats-kinasemediated-ctcf-phosphorylation-and-selective-loss-of-genomic-binding." class="section level3">
<h3>LATS kinase–mediated CTCF phosphorylation and selective loss of genomic binding.</h3>
<p><em>Science Advances</em> . 2020 Feb 19;6(8):eaaw4651. doi: 10.1126/sciadv.aaw4651.</p>
<p>N/A</p>
<p>2020</p>
<p>Luo H, Yu Q, Liu Y, <strong>Tang M</strong>, Liang M, Zhang D, Xiao T, Wu L, Tan M, Ruan Y, Bungert J, Lu J.</p>
</div>
<div id="fast-analysis-of-scatac-seq-data-using-a-predefined-set-of-genomic-regions." class="section level3">
<h3>Fast analysis of scATAC-seq data using a predefined set of genomic regions.</h3>
<p><strong>F1000Research</strong>. 2020, 9:199 (<a href="https://doi.org/10.12688/f1000research.22731.2" class="uri">https://doi.org/10.12688/f1000research.22731.2</a>)</p>
<p>N/A</p>
<p>2020</p>
<p>Valentina Giansanti, <strong>Ming Tang</strong>, Davide Cittaro.</p>
</div>
<div id="kmt2d-deficiency-impairs-super-enhancers-to-confer-a-glycolytic-vulnerability-in-lung-cancer." class="section level3">
<h3>KMT2D Deficiency Impairs Super-Enhancers to Confer a Glycolytic Vulnerability in Lung Cancer.</h3>
<p><em>Cancer Cell</em>. 2020 Apr 13;37(4):599-617.e7. doi: 10.1016/j.ccell.2020.03.005.</p>
<p>N/A</p>
<p>2020</p>
<p>Alam H*, <strong>Tang M*</strong>, Maitituoheti M, Dhar S, Kumar M, han C, Ambati C, Amin S, Gu B, Chen T, Lin Y,
Chen J, Muller F, Putluri N, Flores E, Demayo F, Baseler L, Rai K, Lee M.</p>
</div>
<div id="tumor-microenvironment-remodeling-enables-bypass-of-oncogenic-kras-dependency-in-pancreatic-cancer" class="section level3">
<h3>Tumor microenvironment remodeling enables bypass of oncogenic KRAS dependency in pancreatic cancer</h3>
<p><em>Cancer Discovery</em>. 2020 Apr 27. doi: 10.1158/2159-8290.CD-19-0597.</p>
<p>N/A</p>
<p>2020</p>
<p>Hou P, Kapoor A, Zhang Q, Li J, Wu C, Li J, Lan Z, <strong>Tang M</strong>, Ma X, Ackroyd J, Kalluri R, Zhang J, Jiang S,
Spring D, Wang Y, DePinho R.</p>
</div>
</div>
<div id="preprint-for-co-first-author" class="section level2" data-icon="file">
<h2>Preprint (* for co-first author)</h2>
<div id="bivalent-and-broad-chromatin-domains-regulate-pro-metastatic-drivers-in-melanoma." class="section level3">
<h3>Bivalent and Broad Chromatin Domains Regulate Pro-metastatic Drivers in Melanoma.</h3>
<p><em>BioRxiv</em> 2019 doi: <a href="https://doi.org/10.1101/721480" class="uri">https://doi.org/10.1101/721480</a></p>
<p>N/A</p>
<p>2019</p>
<p>Terranova C*, <strong>Tang M*</strong>, Maitituoheti M, Raman A, Schulz J, Amin S, Orouji E, Tomczak K, Sarkar S, Oba J,
Crasy C, Wu C, Zhao D, Chen K, Haydu L, Wang W, Lazar A, Woodman S, Bernatchez C, Rai K.</p>
</div>
<div id="evaluating-single-cell-cluster-stability-using-the-jaccard-similarity-index." class="section level3">
<h3>Evaluating single-cell cluster stability using the Jaccard similarity index.</h3>
<p><em>BioRxiv</em> 2020 doi: <a href="https://doi.org/10.1101/2020.05.26.116640" class="uri">https://doi.org/10.1101/2020.05.26.116640</a></p>
<p>N/A</p>
<p>2020</p>
<p><strong>Tang M</strong>, Kaymaz Y, Logeman B, Eichhorn S, Liang Z, Dulac C, Sackton T.</p>
</div>
</div>
<div id="book-chapters" class="section level2" data-icon="book">
<h2>Book Chapters</h2>
<div id="genetically-altered-cancer-epigenome." class="section level3">
<h3>Genetically altered cancer epigenome.</h3>
<p>Epigenetic Gene Expression and Regulation. 2016. Elsevier, Inc. Edited by
Suming Huang, Michael Litt, and Cynthia Ann Blakey</p>
<p>N/A</p>
<p>2016</p>
<p><strong>Tang M</strong>, Luo HC and Lu J</p>
</div>
<div id="the-biostar-handbook-a-beginners-guide-to-bioinformatics." class="section level3">
<h3>The biostar Handbook: A Beginner’s Guide to Bioinformatics.</h3>
<p><a href="https://read.biostarhandbook.com/" class="uri">https://read.biostarhandbook.com/</a> 2017. Edited by Istvan Albert.</p>
<p>N/A</p>
<p>2017</p>
<p>Sebastian A, Albert R, Leipzig J, Kelkar H, <strong>Tang M</strong>, Shen W, Coster WD</p>
</div>
<div id="computational-analysis-of-epigenetic-modifications-in-melanoma-cancer" class="section level3">
<h3>Computational analysis of epigenetic modifications in melanoma cancer</h3>
<p>Computational Epigenetics and Diseases. 2017. Elsevier,Inc</p>
<p>N/A</p>
<p>2018</p>
<p>Rai K, <strong>Tang M</strong>.</p>
</div>
</div>
<div id="software" class="section level2" data-icon="laptop">
<h2>Software</h2>
<div id="pyflow-chipseq-a-snakemake-workflow-for-processing-chip-seq-data." class="section level3">
<h3>pyflow-ChIPseq, a snakemake workflow for processing ChIP-seq data.</h3>
<p>github link <a href="https://github.com/crazyhottommy/pyflow-ChIPseq" class="uri">https://github.com/crazyhottommy/pyflow-ChIPseq</a></p>
<p>N/A</p>
<p>2017</p>
</div>
<div id="pyflow-atacseq-a-snakemake-workflow-for-processing-atac-seq-data." class="section level3">
<h3>pyflow-ATACseq, a snakemake workflow for processing ATAC-seq data.</h3>
<p>github link <a href="https://github.com/crazyhottommy/pyflow-ATACseq" class="uri">https://github.com/crazyhottommy/pyflow-ATACseq</a></p>
<p>N/A</p>
<p>2017</p>
</div>
<div id="pyflow-rnaseq-a-snakemake-workflow-for-processing-rna-seq-data." class="section level3">
<h3>pyflow-RNAseq, a snakemake workflow for processing RNA-seq data.</h3>
<p>github link <a href="https://github.com/crazyhottommy/pyflow-RNAseq" class="uri">https://github.com/crazyhottommy/pyflow-RNAseq</a></p>
<p>N/A</p>
<p>2017</p>
</div>
<div id="pyflow-dnaseq-a-snakemake-workflow-for-processing-dna-seq-data." class="section level3">
<h3>pyflow-DNAseq, a snakemake workflow for processing DNA-seq data.</h3>
<p>gitlab link <a href="https://gitlab.com/tangming2005/snakemake_DNAseq_pipeline/tree/multiRG" class="uri">https://gitlab.com/tangming2005/snakemake_DNAseq_pipeline/tree/multiRG</a></p>
<p>N/A</p>
<p>2018</p>
</div>
<div id="pyflow-rrbs-a-snakemake-workflow-for-processing-rrbs-data." class="section level3">
<h3>pyflow-RRBS, a snakemake workflow for processing RRBS data.</h3>
<p>gitlab link <a href="https://gitlab.com/tangming2005/pyflow-RRBSeq" class="uri">https://gitlab.com/tangming2005/pyflow-RRBSeq</a></p>
<p>N/A</p>
<p>2018</p>
</div>
<div id="pyflow-scatacseq-a-snakemake-workflow-for-post-processing-single-cell-atac-seq-data." class="section level3">
<h3>pyflow-scATACseq, a snakemake workflow for post-processing single-cell ATAC-seq data.</h3>
<p>github link <a href="https://github.com/crazyhottommy/pyflow-scATACseq" class="uri">https://github.com/crazyhottommy/pyflow-scATACseq</a></p>
<p>N/A</p>
<p>2019</p>
</div>
<div id="scatacutils-an-r-package-for-analyzing-single-cell-atac-seq-data." class="section level3">
<h3>scATACutils, an R package for analyzing single-cell ATAC-seq data.</h3>
<p>github link <a href="https://github.com/crazyhottommy/scATACutils" class="uri">https://github.com/crazyhottommy/scATACutils</a></p>
<p>N/A</p>
<p>2019</p>
</div>
<div id="scclusteval-an-r-package-for-evaluating-single-cell-rna-seq-cluster-stability" class="section level3">
<h3>scclusteval, an R package for evaluating single-cell RNA-seq cluster stability</h3>
<p>github link <a href="https://github.com/crazyhottommy/scclusteval" class="uri">https://github.com/crazyhottommy/scclusteval</a></p>
<p>N/A</p>
<p>2019</p>
</div>
</div>
<div id="teaching-experience" class="section level2" data-icon="chalkboard-teacher">
<h2>Teaching Experience</h2>
<div id="software-carpentry-2-day-workshop." class="section level3">
<h3>Software Carpentry 2-day workshop.</h3>
<p>Instructor of R and bash for the software carpentry workshop held in the University of Miami. <a href="https://divingintogeneticsandgenomics.rbind.io/talk/2015-miami-carpentry/" class="uri">https://divingintogeneticsandgenomics.rbind.io/talk/2015-miami-carpentry/</a></p>
<p>FL, USA</p>
<p>2014</p>
</div>
<div id="gs01-1143-introduction-to-bioinformatics-course." class="section level3">
<h3>GS01 1143 Introduction to Bioinformatics course.</h3>
<p>I taught the ChIP-seq bioinformatics lab for the GS01 1143 Introduction to Bioinformatics course to ~30 people with diverse background in MD Anderson Cancer Center. <a href="https://divingintogeneticsandgenomics.rbind.io/talk/2017-gsbs-chipseq-class/" class="uri">https://divingintogeneticsandgenomics.rbind.io/talk/2017-gsbs-chipseq-class/</a></p>
<p>TX, USA</p>
<p>2017</p>
</div>
<div id="next-gen-sequence-analysis-workshop." class="section level3">
<h3>Next-Gen Sequence Analysis Workshop.</h3>
<p>I taught the ChIP-seq lesson for 2018 ANGUS Next-Gen Sequence Analysis Workshop held in UC Davis from 7/1/2018 to 7/14/2018, and TAed for the rest of the sessions. <a href="https://divingintogeneticsandgenomics.rbind.io/talk/2018-dibsi-course/" class="uri">https://divingintogeneticsandgenomics.rbind.io/talk/2018-dibsi-course/</a></p>
<p>CA, USA</p>
<p>2018</p>
</div>
<div id="harvard-fas-informatics-nanocourse" class="section level3">
<h3>Harvard FAS informatics nanocourse</h3>
<p>In this 2-week long Harvard FAS informatics nanocourse, I co-taught snakemake for one afternoon and lead-instructed scRNAseq analysis for a full day. <a href="https://divingintogeneticsandgenomics.rbind.io/talk/2019-harvard-fas-workshop/" class="uri">https://divingintogeneticsandgenomics.rbind.io/talk/2019-harvard-fas-workshop/</a></p>
<p>MA, USA</p>
<p>2019</p>
</div>
<div id="liu-lab-new-hire-bioinformatics-training" class="section level3">
<h3>Liu Lab new hire bioinformatics training</h3>
<p>I TAed a 5-week long bioinformatics training for the Liu lab new hires using materials from Harvard course STAT115/215: Introduction to Computational Biology and Bioinformatics</p>
<p>MA, USA</p>
<p>2020</p>
</div>
</div>
<div id="invited-talk" class="section level2" data-icon="microphone-alt">
<h2>Invited talk</h2>
<div id="from-cell-line-to-command-line-my-journey-to-bioinformatics" class="section level3">
<h3>From cell line to command line: my journey to bioinformatics</h3>
<p>I was invited to give a talk to my PhD Genetics and Genomics program at the University of Florida. I talked about my personal experience on how I started doing bioinformatics and shared tips and resources to get the students started their own journey into bioinformatics. <a href="https://divingintogeneticsandgenomics.rbind.io/talk/2018-uf-talk/" class="uri">https://divingintogeneticsandgenomics.rbind.io/talk/2018-uf-talk/</a></p>
<p>FL, USA</p>
<p>2018</p>
</div>
<div id="reproducible-research-in-bioinformatics" class="section level3">
<h3>Reproducible research in bioinformatics</h3>
<p>I was invited to give a talk on reproducible bioinformatics research to the students in the Bunker Hill Community College in Boston. <a href="https://divingintogeneticsandgenomics.rbind.io/talk/2019-bunkerhill-talk/" class="uri">https://divingintogeneticsandgenomics.rbind.io/talk/2019-bunkerhill-talk/</a></p>
<p>MA, USA</p>
<p>2019</p>
</div>
<div id="reproducible-computing-for-your-own-benifit." class="section level3">
<h3>Reproducible computing for your own benifit.</h3>
<p>I was invited to give a talk on reproducible computing at Center for Functional Cancer Epigenetics(CFCE) in Dana-Farber Cancer Institute.</p>
<p>MA, USA</p>
<p>2020</p>
</div>
</div>
<div id="professional-service" class="section level2" data-icon="pen-alt">
<h2>Professional service</h2>
<div id="reviewers-for-journals" class="section level3">
<h3>Reviewers for journals</h3>
<p>N/A</p>
<p>N/A</p>
<div class="concise">
<ul>
<li>Clinical Cancer Research.</li>
<li>GigaScience.</li>
<li>Bioinformatics</li>
<li>Aging</li>
<li>Cancer Research</li>
<li>F1000 Research</li>
<li>Cancer Letters</li>
<li>Cell Reports</li>
<li>BMC genetics</li>
<li>Oncogene</li>
<li>PLOS ONE</li>
<li>Oncotarget</li>
</ul>
</div>
</div>
</div>
<div id="professional-membership" class="section level2" data-icon="suitcase">
<h2>Professional membership</h2>
<div id="member-society-for-neuro-oncology-sno" class="section level3">
<h3>Member, Society for Neuro-Oncology (SNO)</h3>
<p>N/A</p>
<p>N/A</p>
<p>2016</p>
</div>
<div id="genomics-advisory-committee-member-for-data-carpentry-usa" class="section level3">
<h3>Genomics Advisory committee member for Data Carpentry, USA</h3>
<p>N/A</p>
<p>N/A</p>
<p>2017</p>
</div>
<div id="certified-instructor-software-carpentry-data-carpentry-usa" class="section level3">
<h3>Certified Instructor, Software Carpentry, Data Carpentry, USA</h3>
<p>N/A</p>
<p>N/A</p>
<p>2017 -</p>
</div>
<div id="member-american-society-of-clinical-oncology-asco" class="section level3">
<h3>Member, American Society of Clinical Oncology (ASCO)</h3>
<p>N/A</p>
<p>N/A</p>
<p>2017</p>
</div>
<div id="member-bioinformatics-organization." class="section level3">
<h3>Member, Bioinformatics Organization.</h3>
<p>N/A</p>
<p>N/A</p>
<p>2020</p>
</div>
</div>
</div>

<script>
(function() {
  var i, s, j, el, els;
  var create_el = function(type, className, content, isHTML) {
    var el = document.createElement(type);
    if (className) el.className = className;
    if (content) {
      if (isHTML) {el.innerHTML = content;} else {el.innerText = content;}
    }
    return el;
  }

  // replace h2 title with h1
  var title = document.getElementById('title').querySelector('h2');
  title.parentNode.replaceChild(create_el('h1', false, title.innerHTML, true), title);

  // add the class .aside to #aside
  el = document.getElementById('aside');
  if (el) el.className += ' aside';

  // tweak class names of sections, and add icons
  els = document.getElementById('main').querySelectorAll('.level2');
  for (i = 0; i < els.length; i++) {
    el = els[i];
    if (i === 0 && el.id === 'title') continue;
    el.className += ' main-block';
    if (el.dataset['concise']) el.className += ' concise';
    // if there is no icon, add an icon:
    if (el.firstElementChild.firstChild && el.firstElementChild.firstChild.nodeName !== 'I') {
      s = el.dataset['icon'] || 'bookmark';
      el.firstElementChild.insertBefore(create_el('i', 'fa fa-' + s), el.firstElementChild.firstChild);
    }
  }

  // tweak class names of blocks in sections
  els = document.getElementById('main').querySelectorAll('.level3');
  for (i = 0; i < els.length; i++) {
    el = els[i];
    if (el.parentNode.id === 'title') continue;
    el.className += ' blocks';
    el.innerHTML = '<div class="date"></div>' + '<div class="decorator"></div>' +
      '<div class="details"><header></header><div></div></div>' + el.innerHTML;
    var ps = el.querySelectorAll('p');
    // move the date paragraph to the date div
    if (ps.length >= 3) {
      s = ps[2].innerText;
      s = s === 'N/A' ? [] : s.split(' - ');
      el.removeChild(ps[2]);
      for (j = 0; j < s.length && j < 2; j++) {
        el.children[0].appendChild(create_el('span', false, s[j]))
      }
    }
    // move title, description, location to the details div
    (function(h) {
      h.appendChild(el.children[3]);
      var p = el.children[3];  // description
      if (p.innerText !== 'N/A') {
        h.appendChild(create_el('span', 'place', p.innerHTML, true));
      }
      el.removeChild(p);
      p = el.children[3];  // location
      if (p.innerText !== 'N/A') {
        s = create_el('span', 'location', p.innerHTML, true);
        s.innerHTML = '<i class="fa fa-map-marker-alt"></i> ' + s.innerHTML;
        h.appendChild(s);
      }
      el.removeChild(p);
    })(el.children[2].firstElementChild);
    // move the rest of content in a block into the last div of the details div
    s = el.children[2];
    while (j = s.nextSibling) {
      if (j.className === 'aside') { s = j; continue; }
      el.children[2].lastElementChild.append(j);
    }
  }
})();
</script>


  </body>
</html>
